IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinUtilBase+Internal.h

```diff
@@ -33,6 +33,12 @@
                                              context:(nullable id<MSIDRequestContext>)context
                                                error:(NSError*__nullable*__nullable)error;
 
++ (NSString *_Nullable)getWPJStringDataFromV2ForTenantId: (NSString *_Nullable)tenantId
```

Comment:
nit: There is an extra space after `:`
```suggestion
+ (NSString *_Nullable)getWPJStringDataFromV2ForTenantId:(NSString *_Nullable)tenantId
```